ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -436,15 +441,21 @@ public boolean maybeFailedDueToOfflineReplicas() {
       routerMetrics.failedMaybeDueToOriginatingDcOfflineReplicasCount.inc();
       return true;
     }
-    if (hasFailedOnCrossColoNotFound() && allReplicaCount - totalNotFoundCount >= replicaSuccessTarget
```

Comment:
Currently only delivery cluster enable "routerUnavailableDueToOfflineReplicas".
Need enable it in EI to test this change.